src/quo2/components/network_amount.cljs

```diff
@@ -0,0 +1,85 @@
+(ns quo2.components.network-amount
+  (:require [clojure.string :as string]
+            [quo.react-native :as rn]
+            [quo2.components.icon :as icon]
+            [quo2.components.text :as text]
+            [quo2.foundations.colors :as colors]
+            [status-im.i18n.i18n :as i18n]))
+
+(def ^:private sizes
+  {:icon {:small 12 :medium 16 :large 20}
+   :container {:small 32 :medium 32 :large 48}
+   :label {:small :label :medium :paragraph-2 :large :paragraph-1}
+   :amount-text {:small :paragraph-2 :medium :paragraph-1 :large :heading-2}
+   :divider {:small {:width 1 :height 8}
+             :medium {:width 1 :height 10}
+             :large {:width 2 :height 12}}})
+
+(defn- format-eth-value [eth-value]
+  (str eth-value \space (i18n/label :t/eth)))
+
+(defn- amount-text [size eth-value]
+  (let [opts {:weight :medium
+              :size (get-in sizes [:amount-text size])
+              :style {:color (colors/theme-colors colors/black colors/white)}}]
+    [text/text opts (format-eth-value eth-value)]))
+
+(defn- label [size & text]
+  (let [opts {:weight :medium
+              :size (get-in sizes [:label size])
+              :style {:color colors/neutral-50}}]
+    [text/text opts (string/join \space text)]))
+
+(defn- divider [size]
+  (let [{:keys [width height]} (get-in sizes [:divider size])]
+    [rn/view {:style {:border-right-width width
+                      :border-right-color (colors/theme-colors colors/neutral-40 colors/neutral-50)
+                      :padding-left 8
+                      :align-self :center
+                      :height height}}]))
+
+(defn- rounded-container [size style & components]
+  (let [container-size (get-in sizes [:container size])
+        default-style {:accessibility-label :network-amount
+                       :background-color (colors/theme-colors colors/white colors/neutral-95)
+                       :border-radius (/ container-size 2)
+                       :padding 6}
+        opts {:style (merge default-style style)}]
+    (into [rn/view opts] components)))
+
+(defn- row [& components]
+  (into [rn/view {:style {:flex-direction :row
+                          :align-items :center}}]
+        components))
+
+(defn- icon-padding-top [size]
+  (if (= size :large) 1 3))
+
+(defn- icon-column [size icon]
+  (let [icon-size (get-in sizes [:icon size])]
+    [rn/view {:flex-direction :column
+              :align-self :flex-start
+              :padding-top (icon-padding-top size)
+              :padding-right 4}
+     [icon/icon icon {:size icon-size :color :none}]]))
+
+(defn- amount-column [size eth-value network-name]
+  [rn/view
+   [row
+    [amount-text size eth-value]
+    [divider size]]
+   [label size (i18n/label :t/on) network-name]])
+
+(defn view
```

Comment:
it's better to name it `network-amount`

Reply:
I thought `view` is the somewhat better name, because `network-amount` reeked like a smurf naming convention to me. Naming it `network-amount/view` is not consistent with the other components though, and when requiring only the component and not the namespace, naming the component function also `network-amount` is clearer.

 I'll change it to `network-amount`.

Reply:
Done.